src/backend/distributed/executor/multi_utility.c

```diff
@@ -37,6 +37,7 @@
 #include "commands/prepare.h"
 #include "distributed/citus_ruleutils.h"
 #include "distributed/colocation_utils.h"
+#include "distributed/foreign_constraint.h"
 #include "distributed/intermediate_results.h"
```

Comment:
Don't we need to remove some of the header includes given that we're removing lots of C code?